data/remnant/remnant 2 cognizance.txt

```diff
@@ -1183,6 +1183,8 @@ mission "Remnant: Cognizance 25"
 		dialog "You have destroyed the ships that the Palavret left behind."
```

Comment:
```suggestion
		dialog "You have destroyed the ships that the Palavret left behind. Time to finish your patrol of the region."
```

Reply:
You could finish the patrol without killing the chasers, however the purpose of this PR is to make sure that doesn’t happen by making it a little clearer.